packages/react-devtools-shared/src/backend/console.js

```diff
@@ -209,7 +229,31 @@ export function patch({
           debugger;
         }
 
-        originalMethod(...args);
+        if (isInStrictMode) {
+          if (!consoleSettingsRef.hideConsoleLogsInStrictMode) {
+            if (isNode) {
+              originalMethod('\x1b[2m%s\x1b[0m', format(...args));
+            } else {
+              let color;
+              switch (method) {
+                case 'warn':
+                  color = 'rgba(250, 180, 50, .5)';
+                  break;
+                case 'error':
+                  color = 'rgba(250, 123, 130, .5)';
+                  break;
+                case 'log':
+                default:
+                  color = 'rgba(125, 125, 125, 0.5)';
+                  break;
+              }
+
+              originalMethod(`%c${format(...args)}`, `color: ${color}`);
```

Comment:
We know this because we're in strict mode (`isInStrictMode`) and the developer didn't hide the console logs (`!consoleSettingsRef.hideConsoleLogsInStrictMode`), which means we'll show the console logs with dimmed colors